SU-HSDO/suhumsci
# SHS-5114: Fix warbler hs-button color

# READY FOR REVIEW

## Summary
- Update link styles to prevent hs-button color to be overridden in warbler

## Need Review By (Date)
11/9

## Urgency
medium

## Steps to Test
1. Install `islamicstudies` site
2. Go to the [Past Event Recordings view](http://islamicstudies.suhumsci.loc/admin/structure/views/view/custm_islamic_events/edit/block_1) config page. Go "Fields > Content: Event Video > Style Settings" and change the "CSS Class" from `hs-secondary-button` to `hs-button`. Save the view
3. Visit the [Past Event Recordings page](http://islamicstudies.suhumsci.loc/news-events/past-event-recordings) and confirm that the "Watch the Video" buttons text is white.

<img width="1000" alt="Screenshot 2023-11-08 at 22 24 30" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/7de8c72d-1997-43b1-92b1-bde47dc76d75">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
